src/Astro/Astro.Autorest/internal/New-AzAstroOrganization.ps1

```diff
@@ -107,6 +107,15 @@ param(
     # Type of managed service identity (where both SystemAssigned and UserAssigned types are allowed).
     ${IdentityType},
 
+    [Parameter(ParameterSetName='CreateExpanded')]
+    [Microsoft.Azure.PowerShell.Cmdlets.Astro.Category('Body')]
+    [Microsoft.Azure.PowerShell.Cmdlets.Astro.Runtime.Info(PossibleTypes=([Microsoft.Azure.PowerShell.Cmdlets.Astro.Models.IUserAssignedIdentities]))]
+    [System.Collections.Hashtable]
+    # The set of user assigned identities associated with the resource.
+    # The userAssignedIdentities dictionary keys will be ARM resource ids in the form: '/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ManagedIdentity/userAssignedIdentities/{identityName}.
+    # The dictionary values can be empty objects ({}) in requests.
+    ${IdentityUserAssignedIdentity},
```

Comment:
The name is changed from `UserAssignedIdentity` to `IdentityUserAssignedIdentity`. Is it expected? @lijinpei2008

Reply:
Yes, the name of the parameter we expose to the user is UserAssignedIdentity, and the PS1 script file that appears in the Internal folder belongs to an intermediate file, in which the contents are not directly displayed to the user, but rather the file is copied to the Custom folder and then modified in some way.